### DISCUSSION TITLE
Add DbUpdateExceptionDestructurer via JSON configuration

### DISCUSSION THREAD

**Discussion by alexdresko** [RehanSaeed/Serilog.Exceptions#170] [Q&A]
I know it's possible to add the exception details enricher using the [JSON configuration plugin](https://github.com/serilog/serilog-settings-configuration), but I can't figure out how to convert [these DbUpdateExceptionDestructurer instructions](https://github.com/RehanSaeed/Serilog.Exceptions#serilogexceptionsentityframeworkcore) to a JSON configuration.

**Comment by Tim-Hodge** [comment]
@alexdresko did you find a solution for this in JSON?

**Comment by RehanSaeed** [comment]
I also went through a phase of trying to configure a lot of Serilog using JSON but came to the conclusion that it's not really possible, then switched to doing some things in code and others in JSON. In our case, I'm not sure we could do it easily, sorry!

**Comment by alexdresko** [comment]
For the record, I also created [Disable programmatic configuration via JSON configuration · Issue #204 · serilog/serilog-settings-configuration](https://github.com/serilog/serilog-settings-configuration/issues/204), wherein the answer was no.
